lib/schema-fs-loader.js

```diff
@@ -115,8 +123,96 @@ const loadAndCreate = (dir, context, callback) => {
   }
 };
 
+const getSortOrder = file => {
+  if (file.includes('.')) {
+    return 0;
+  }
+  if (file.endsWith('.category')) {
```

Comment:
This statement will never execute because file will have `.` and function will exit with the first statement.